src/strings/unicode.h

```diff
@@ -16,6 +16,11 @@ struct MVMUnicodeNameRegistry {
     MVMGrapheme32 codepoint;
     UT_hash_handle hash_handle;
 };
+struct MVMUnicodeGraphemeNameRegistry {
+    char *name;
+    MVMint32 structitem;
```

Comment:
Not entirely keen on `structitem` as a name...it seems more of an index to the codepoint sequence?